components/Reviews.tsx

```diff
@@ -0,0 +1,130 @@
+import { Button, Card, Select, Text, User } from '@geist-ui/react';
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { STORY_REVIEW_STATUS, STORY_REVIEW_TYPES } from '../constants';
+import { getPeople } from '../redux/selectors/projects.selectors';
+import { Review } from '../redux/types';
+
+interface ReviewParams {
+  reviewsData: Review[];
```

Comment:
I'd prefer this named to `reviews`, wdyt?

Reply:
I agree